supporting_scripts/hyperion/consistency-check-benchmark/course.py

```
@@ -223,14 +227,20 @@ def get_exercise_ids_request(session: requests.Session, course_id: int) -> Dict[
             if ex_title is not None and ex_id is not None:
                 exercises_map[ex_title] = ex_id
 
-        return __transform_exercise_json_keys(exercises_map)
+        logging.debug(f"Raw exercise titles from server: {list(exercises_map.keys())}")
+        transformed = __transform_exercise_json_keys(exercises_map)
+        logging.info(f"Retrieved {len(transformed)} programming exercise(s) for the course.")
+        if not transformed:
+            logging.error("Step 11 failed: No exercise IDs returned — ensure exercises are imported via Steps 9–10 in exercises.py, then execute Step 11")
```

Ruff (0.15.5)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)